reference/JsBeforeCollectCallback.md

```diff
@@ -4,4 +4,4 @@ A callback called before collection.
 typedef void (CALLBACK *JsBeforeCollectCallback)(_In_opt_ void *callbackState);
 ```
 ### Remarks 
-Use **JsSetBeforeCollectCallback<** to register this callback.
\ No newline at end of file
+Use **JsSetRuntimeBeforeCollectCallback<** to register this callback.
```

Comment:
While we're here, what is that `<` doing here anyway? Is that a typo or some convention I'm not familiar with?